File: CursoCsharpUdemy/exer/outros/lista64Exer/Exer02.cs
/***********************************
*   2) Faça um programa que:
*   - Leia a cotação do dólar
*   - Leia um valor em dólares
*   - Converta esse valor para Real
*   - Mostre o resultado
************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer02
    {
        static void Main(string[] args)
        {
            double dollar, salaryDollar, salaryReal;

            Console.WriteLine("Informe a cotação do dolar atual: ");
            dollar = double.Parse(Console.ReadLine());

            Console.WriteLine("Informe seu sálario em dolar: ");
            salaryDollar = double.Parse(Console.ReadLine());

            salaryReal = salaryDollar * dollar;

            Console.Write("Seu sálario em real será: " + salaryReal.ToString("F2"));
        }
    }
}

File: CursoPython3/exer/lista64Exer/exer02.py
"""
2) Faça um programa que: 
- Leia a cotação do dólar 
- Leia um valor em dólares 
- Converta esse valor para Real 
- Mostre o resultado 
"""

exchange_rate = float(input("Digite a cotação atual do dólar: "))
dollar_value = float(input("Digite o valor que você quer converter: "))

real_value = dollar_value * exchange_rate

print("O valor convertido para real é R$ {:.2f}".format(real_value))
